src/services/ethgasstation/actions.js

```diff
@@ -0,0 +1,13 @@
+const ETH_GAS_STATION = process.env.ETH_GAS_STATION
+
+export const getGasStationStatus = async () => {
+	const result = await fetch(`${ETH_GAS_STATION}json/ethgasAPI.json`, {

```

Comment:
Isn't it better to use `provider.getGasPrice()`
You can make `gas` service instead `ethgasstation` because if we want to change the gas price info provider it will not be enough to change `ETH_GAS_STATION` env prop. 
Make it gas service and use common method names not "ethgasstation" related

Reply:
it's better

I forgot about it when specifying the issue